ecosystem/indexer-grpc/indexer-grpc-table-info/src/backup_restore/gcs.rs

```diff
@@ -176,6 +176,17 @@ impl GcsBackupRestoreOperator {
             snapshot_tar_file_name = snapshot_tar_file_name.as_str(),
             "[Table Info] Starting to compress the folder.",
         );
+        // If target path does not exist, wait and log.
+        if !snapshot_path.exists() {
+            aptos_logger::warn!(
+                snapshot_path = snapshot_path.to_str(),
+                snapshot_tar_file_name = snapshot_tar_file_name.as_str(),
+                epoch = epoch,
+                "[Table Info] Directory does not exist. Waiting for the directory to be created."
+            );
+            std::thread::sleep(std::time::Duration::from_secs(30));
```

Comment:
should we not use tokio sleep here? so we're not blocking whole thread

Reply:
good call; done.